pkg/products/monitoringspec/reconciler.go

```diff
@@ -380,7 +380,7 @@ func (r *Reconciler) reconcileRoleBinding(ctx context.Context,
 		}
 		roleBinding.RoleRef = rbac.RoleRef{
 			APIGroup: roleRefAPIGroup,
-			Kind:     bundle.RoleKind,
+			Kind:     "Role",
```

Comment:
I see these constants still exist in the latest version of the operator-framework package, any reason for opting for manually defined string instead? https://pkg.go.dev/github.com/operator-framework/operator-registry@v1.28.0/pkg/lib/bundle#pkg-constants

Reply:
Yes, this is a workaround to prevent - https://github.com/operator-framework/operator-registry/issues/1114

Reply:
Aha gotcha thanks